src/Mod/MeshPart/Gui/MeshFlatteningCommand.py

```diff
@@ -69,10 +73,12 @@ def IsActive(self):
 class CreateFlatFace(BaseCommand):
     """create a flat face from a single face
        only full faces are supported right now"""
-       
+
     def GetResources(self):
-        return {'Pixmap': 'MeshPart_CreateFlatFace.svg', 'MenuText': 'Unwrap Face', 'ToolTip': 'find a flat representation of a mesh'}
-    
+        return {'Pixmap': 'MeshPart_CreateFlatFace.svg',
+                'MenuText': QT_TRANSLATE_NOOP("MeshPart_FlatteningCommand", "Unwrap Face"),
+                'ToolTip': QT_TRANSLATE_NOOP("MeshPart_FlatteningCommand", "Find a flat representation of a mesh.")}
```

Comment:
@yorikvanhavre, based on what loooo wrote (https://github.com/FreeCAD/FreeCAD/pull/5028#issuecomment-966509235), I think we need to change the description from `mesh` to `face`